metrics/tarantool/event_loop.lua

```diff
@@ -0,0 +1,43 @@
+local clock = require('clock')
+local fiber = require('fiber')
+local metrics = require('metrics')
+
+local collectors_list = {}
+
+local function monitor()
+    fiber.self():name("tx_speed")
+
+    local max = -math.huge
+
+    while true do
```

Comment:
Please avoid `while true` pattern as aggressive snippet.
Detox it with something else

@ochaton example

https://gist.github.com/ochaton/6bc9a7030df7b4e7c63703599a5b2f96